### PR TITLE
feat: implement heartbeat api in ingestion

### PR DESCRIPTION
**Pull request by dengmingtong**
----

*By submitting this pull request, I confirm that my contribution is made under the terms of the Apache-2.0 license*

## Summary

Add api for return server timestamp

## Implementation highlights

Add api for return server timestamp
Add UT

## Test checklist

- [ ] add new test cases
- [x] all code changes are covered by unit tests
- [x] end-to-end tests
  - [ ] deploy web console with CloudFront + S3 + API gateway
  - [ ] deploy web console within VPC
  - [x] deploy ingestion server
    - [ ] with MSK sink
    - [ ] with KDS sink
    - [x] with S3 sink
  - [ ] deploy data processing
  - [ ] deploy data modeling
    - [ ] new Redshift Serverless
    - [ ] provisioned Redshift
    - [ ] Athena
  - [ ] deploy with reporting

## Is it a breaking change

- [ ] add parameters without default value in stack
- [ ] introduce new service permission in stack
- [ ] introduce new top level stack module

## Miscellaneous

- [ ] introduce new symbol link source file(s) to be shared among infra code, web console frontend, and web console backend

### CODE REVIEW

File: src/ingestion-server/server/images/nginx/config/nginx.conf [hunk 1]
```diff
@@ -75,6 +75,34 @@ http {
             proxy_pass http://127.0.0.1:8685;
         }
 
+        location %%PING_ENDPOINT_PATH%% {
+            access_log off;
+
+            add_header Content-Type text/plain;
```
Comment (zxkane) [inline src/ingestion-server/server/images/nginx/config/nginx.conf:81]: Let's use JSON for supporting other attributes in the future.

**Review by zxkane — CHANGES_REQUESTED** [review]
- [x] add test for ecs configuration changes

File: src/ingestion-server/server/images/nginx/config/nginx.conf (hunk 1 above)
Comment (dengmingtong) [reply src/ingestion-server/server/images/nginx/config/nginx.conf]: Fixed

**Comment by dengmingtong** [comment]
> * [x]  add test for ecs configuration changes

Added

**Review by zxkane — APPROVED** [review]

**Review by zxkane — APPROVED** [review]

**Pull request merged**